pkgs/development/tools/rust/sqlx-cli/default.nix

```diff
@@ -20,6 +20,12 @@ rustPlatform.buildRustPackage rec {
   buildInputs = lib.optionals stdenv.isLinux [ openssl ]
     ++ lib.optionals stdenv.isDarwin [ SystemConfiguration CoreFoundation Security libiconv ];
 
+  passthru.tests.version = testVersion {
+    package = sqlx-cli;
+    command = "sqlx --version";
+    inherit version;
```

Comment:
Should work without this. :thinking: 

```suggestion
```

Reply:
Removed as of 61e1418f8affb9616f71d26c14e01824a722bcde, thanks!